packages/cozy-clisk/src/contentscript/ContentScript.js

```diff
@@ -425,10 +425,25 @@ export default class ContentScript {
    * Send log message to the launcher
    *
    * @param {string} message : the log message
-   * @todo Use cozy-logger to add logging level and other features
+   * @param {string} message: the log message
    */
-  log(message) {
-    this.bridge?.emit('log', message)
+  log(level, message) {
+    const allowedLevels = ['debug', 'info', 'warn', 'error']
+    if (!message) {
+      log.warn(
+        `you are calling log without message, use log(level,message) instead`
+      )
+      return
+    }
+    if (!allowedLevels.includes(level)) {
+      level = 'debug'
+    }
+    const now = new Date()
+    this.bridge?.emit('log', {
+      timestamp: now,
+      level,
+      msg: message
+    })
```

Comment:
Also there is a few call to change: 
https://github.com/konnectors/libs/blob/master/packages/cozy-clisk/src/contentscript/ContentScript.js#L66-L68

for instance

Reply:
Done. I leveled this log in debug due to url information.